ptc/palette.py

```diff
@@ -37,4 +41,7 @@ def __init__(self, palette_name=None, **kwargs):
 
     def load_palette(self, name):
         simple.LoadPalette(paletteName=name)
-        self.server.controller.on_data_change()
+        try:
+            self.server.controller.on_data_change()
```

Comment:
`if self.server.controller.on_data_change.exists():` ?

Reply:
I think it would be nice to inform user that the controller function is not implemented?

Reply:
and in this case I prefer the try/catch pattern - in python in a general maner